### PR TITLE
Add Learn to Program to Misc

### PR DESCRIPTION
**Pull request by karlhorky**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -61,6 +61,14 @@
...
 - [Inspiration](https://github.com/NoahBuscher/Inspire)
 
+## Web development
+
+- [Django](https://github.com/rosarior/awesome-django)
+- [Play1 Framework](https://github.com/PerfectCarl/awesome-play1)
+- [CakePHP](https://github.com/dereuromark/awesome-cakephp)
+- [Symfony2](https://github.com/EmanueleMinotto/awesome-symfony2)
+- [Laravel](https://github.com/tuwannu/awesome-laravel)
+- [Learn to Program](https://github.com/karlhorky/learn-to-program)
```
Comment (sindresorhus) [inline readme.md]: I don't see how `Learn to Program` is specific to Web Development.

Comment (karlhorky) [inline readme.md]: The resources are mostly based on web development and web technologies. The title is maybe not the best. Maybe if we put an annotation in parentheses?

- [Learn to Program](https://github.com/karlhorky/learn-to-program) (Web Dev foundation)

Comment (karlhorky) [inline readme.md]: Or would you add it to another section?

Comment (karlhorky) [inline readme.md]: @sindresorhus The resources are mostly based on web development and web technologies. The title is maybe not the best. Maybe if we put an annotation in parentheses?

- [Learn to Program](https://github.com/karlhorky/learn-to-program) (Web Dev foundation)

Or would you add it to another section?

Comment (sindresorhus) [inline readme.md]: I don't see why you would limit yourself to just Web Development. Learning to program is about a lot more than that. Also from looking at that list a more proper title would be `Back-end developement`, but that should be a separate PR.

So, update this PR to only add your entry to the misc category and do a separate one for the back-end category ;)

Comment (karlhorky) [inline readme.md]: @sindresorhus as to limiting myself, I want to keep the list focused and short. A long wall of links can be daunting to newcomers, who this repository is mainly aimed at.

Why I've chosen this name is simplicity. This is about the audience again. Absolute beginners will understand **Learn to Program** or **Learn to Code** better than **Learn to become a Web Developer**. Secondly, my list is focused less on what concrete skills a person will have afterwards - it's more about:
- awakening interest in programming in general
- some more advanced resources for those already in the industry

That said, you have a valid point - for people already in the industry, it's not clear what this list is offering on first glance. For this reason, I'll add a subheading to the list: `Web Development Foundation` or something.

I've moved my entry to the misc category and removed the other entries. I'll open a separate pull request for them.

Comment (sindresorhus) [inline readme.md]: I don't agree, but it's your list, so fair enough.

Comment (karlhorky) [inline readme.md]: Care to elaborate? I'm open to changing my opinion.
